modules/aws/vendor-access/variables.tf

```diff
@@ -52,10 +52,17 @@ variable "eks_cluster_pattern" {
 }
 
 variable "external_id" {
+  default     = ""
   description = "A external ID that correspond to your Organization within StreamNative Cloud, used for all STS assume role calls to the IAM roles created by the module. This will be the organization ID in the StreamNative console, e.g. \"o-xhopj\"."
   type        = string
 }
 
+variable "external_ids" {
+  default     = []
```

Comment:
It should not have default value, users have to provide the Organization id list as the external ids

Reply:
How should we handle this, I thought the idea would be to allow either the external_id OR the external_ids value to be set. If we remove the `default = []` it will break existing modules when they update because it will expect a defined value.